docs/NATIVE-SEGWIT-UPGRADE.md

```diff
@@ -0,0 +1,112 @@
+# UPGRADING YOUR JOINMARKET TO NATIVE SEGWIT (bech32)
+
+## Motivation: why?
+
+I guess the large majority of Joinmarket users know, but for completeness: native segwit p2wpkh is substantially (perhaps 25% on average?) cheaper than p2sh-p2wpkh which we currently used, and is rapidly becoming industry standard amongst most users wallets and other Bitcoin transaction operators, including other coinjoin systems, e.g. Wasabi - this latter point is very important in that it helps with privacy to join in with the crowd. In particular, Payjoins require both wallets to use the same scriptPubkey (or address) type.
+## Do I need to do anything at all?
```

Comment:
nit - add blank line before header.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/734/commits/8a885aaa999225cfba1aa9314ca5bb308fb2c4fe